src/Acr.UserDialogs.Android/Builders/DatePromptBuilder.cs

```diff
@@ -64,5 +69,12 @@ public static long ToUnixTimestamp(this DateTime dateTime)
             var ms = utc.Subtract(Epoch).TotalMilliseconds;
             return Convert.ToInt64(ms);
         }
+
+        private static bool IsBrokenSamsungDevice()
+        {
+            return Android.OS.Build.Manufacturer.Equals("samsung", StringComparison.OrdinalIgnoreCase) &&
+                   Android.OS.Build.VERSION.SdkInt >= BuildVersionCodes.Lollipop &&
+                   Android.OS.Build.VERSION.SdkInt <= BuildVersionCodes.Lollipop;
```

Comment:
`SdkInt >= Lollipop && SdkInt <= Lollipop` is equal to `SdkInt == Lollipop`.
If something needs to be less than or equal and greater than or equal, the only way to fulfil both is to be equal.